.travis.yml

```diff
@@ -1,7 +1,8 @@
 language: r
+sudo: false
+cache: packages
 
-# R support in Travis CI currently requires sudo http://docs.travis-ci.com/user/languages/r/
-sudo: true
-
-apt_packages:
- - libv8-dev
+addons:
```

Comment:
It's not R specific functionality so it shouldn't be in that documentation.
On Feb 29, 2016 5:40 PM, "Dean Attali" <notifications@github.com> wrote:

> In .travis.yml
> <https://github.com/daattali/shinyjs/pull/60#discussion_r54491849>:
>
> >
> > -# R support in Travis CI currently requires sudo http://docs.travis-ci.com/user/languages/r/
> > -sudo: true
> > -
> > -apt_packages:
> > - - libv8-dev
> > +addons:
>
> @jimhester <https://github.com/jimhester> @craigcitro
> <https://github.com/craigcitro> I just looked at the travis docs for R,
> and this command (addons/apt/packages) is not mentioned there , fyi
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/daattali/shinyjs/pull/60/files#r54491849>.
>